Tribler/community/multichain/database.py

```diff
@@ -39,13 +39,21 @@
 
 CREATE TABLE option(key TEXT PRIMARY KEY, value BLOB);
 INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_DB_VERSION) + u"""');
+
+CREATE INDEX IF NOT EXISTS idx_mc_public_key ON multi_chain(public_key);
+CREATE INDEX IF NOT EXISTS idx_mc_link_public_key ON multi_chain(link_public_key);
 """
 
 upgrade_to_version_2_script = u"""
 DROP TABLE IF EXISTS multi_chain;
 DROP TABLE IF EXISTS option;
 """
 
+create_index_script = u"""
```

Comment:
43-44 adds the indexes when a user has to update the entire database, line 52-53 add the indexes without dropping the entire database first.
43-44 can be deleted, but when the database is changed to a new schema, it is less likely the lines are overseen and when the database is updated, 52-53 can be removed